src/ssl.c

```diff
@@ -24958,6 +25008,32 @@ void wolfSSL_get0_next_proto_negotiated(const WOLFSSL *s, const unsigned char **
 }
 #endif /* HAVE_ALPN */
 
+WOLFSSL_API int wolfSSL_CTX_set1_curves_list(WOLFSSL_CTX* ctx, char* names)
```

Comment:
Why are only 3 curves detected?  Can you add a little documentation to this function please?